internal/provider/client_dashboard.go

```diff
@@ -59,10 +60,11 @@ func (c *SquaredUpClient) GetDashboard(dashboardId string) (*Dashboard, error) {
 	return &newDashboard, nil
 }
 
-func (c *SquaredUpClient) UpdateDashboard(dashboardId string, displayName string, workspaceId string, dashboardContent string) (*Dashboard, error) {
+func (c *SquaredUpClient) UpdateDashboard(dashboardId string, displayName string, workspaceId string, timeframe string, dashboardContent string) (*Dashboard, error) {
 	DashboardPayload := map[string]interface{}{
 		"displayName": displayName,
 		"workspaceId": workspaceId,
```

Comment:
You can't change the workspace of a dashboard so this field is redundant as-is the workspaceId argument to the UpdateDashboard function

Reply:
I will get this updated on my next PR